unikraft/lib/library.go

```diff
@@ -79,26 +79,15 @@ type LibraryConfig struct {
 	// ultimately handled by the packmanager).
 	source string
 
-	// origin contains the URL of the remote source code if this library wraps an
-	// existing library.
-	origin string
-
 	// list of kconfig values specific to this library.
 	kconfig kconfig.KeyValueMap
 
 	// kname the kconfig name which enables this library.
 	kname string
 
-	// projectdir is the root location of the project that this library is a
-	// member of.
-	projectdir string
-
 	// path is the location to this library within the context of a project.
 	path string
 
-	// patchdir is the directory where patches to the origin library are kept.
-	patchdir string
-
```

Comment:
It's future because you know about it and have your own conception of how _you_ want to use them, but as-is it is dead code.

I suggest we comment these fields, not hack-export them. But ideally this code should only exist in a branch where it is being actively worked on as part of a concrete feature.

Reply:
I commented the fields in [4b56ec2a..252bb66f](https://github.com/unikraft/kraftkit/compare/4b56ec2aa2ca80b4cd7183f999ae51917e84504e..252bb66f3ef2e07eca15a5e2b4fa81e354207a60).

But like I mentioned, if the code only exists in one developer's head, it has no place in a shared codebase IMO.

Reply:
Would you mind creating a feature request referring to that part of the code, describing what needs to be done?

Reply:
Created #429

Reply:
Thanks for taking care of this!